src/Desktop/Utility/ExtensionMethods.cs

```diff
@@ -72,6 +76,8 @@ public static ProcessModule GetProcessModule(this A11yElement e)
         /// <returns></returns>
         public static List<DesktopElement> ToListOfDesktopElements( this IUIAutomationElementArray array)
```

Comment:
Not strictly in this PR, but there's an extra space in this line
```suggestion
        public static List<DesktopElement> ToListOfDesktopElements(this IUIAutomationElementArray array)
```